examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,69 @@
+"""
+Extracting brain signal from spheres
+====================================
+
+This example extract brain signals from spheres described by the coordinates
+of their center in MNI space and a given radius in millimeters. In particular,
+this example extracts signals from Default Mode Network regions and compute a
+connectome from them.
+
+"""
+
+# Fetching datasets ###########################################################
+from nilearn import datasets
+adhd_dataset = datasets.fetch_adhd(n_subjects=1)
+
+# print basic information on the dataset
+print('First subject functional nifti image (4D) is at: %s' %
+      adhd_dataset.func[0])  # 4D data
+
+
+# Extracting region signals ###################################################
+from nilearn import input_data
+
+
+# Coordinates of Default Mode Network
+dmn_coords = [(0, -52, 18), (-46, -68, 32), (46, -68, 32), (0, 50, -5)]
+labels = [
+    'Posterior Cingulate Cortex',
+    'Left Temporoparietal junction',
+    'Right Temporoparietal junction',
+    'Medial prefrontal cortex'
+]
+
+masker = input_data.NiftiSpheresMasker(
+    dmn_coords, radius=8,
+    detrend=True, standardize=True,
+    low_pass=0.1, high_pass=0.01, t_r=2.5,
+    memory='nilearn_cache', memory_level=1, verbose=2)
+
+func_filename = adhd_dataset.func[0]
+confound_filename = adhd_dataset.confounds[0]
+
+time_series = masker.fit_transform(func_filename,
+                             confounds=[confound_filename])
+
+
+# Computing group-sparse precision matrices ###################################
```

Comment:
Because covariance estimators works better. I think it's more important to show people things they don't know, rather than showing simple stuff that gives poor results.